Components/Queues/Drivers/AmqpQueueDriver.php

```diff
@@ -165,7 +165,7 @@ public function nack()
         $this->currentEnvelope = null;
     }
 
-    public function createQueueMessage()
+    public function createQueueMessage($queueName = null, $options = [])
```

Comment:
Which is the reason to add these arguments in a method that is not using them?

Reply:
The reason is related to compatibility with some PHP versions like PHP 7.2. This versions requires the same method signature even if the parameters are not used. This was my other solution to remove that "if instance of PHPAMQPLib" from the QueueManager.php